src/main/java/seedu/address/logic/commands/ExportTeamCommand.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.TeamCommandsParser;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+/**
+ * Export team members' details to the user's clipboard.
+ */
+public class ExportTeamCommand extends Command {
+
+    public static final String COMMAND_WORD = "export-team";
```

Comment:
User should have already entered `team` as main command before this command, so this seems redundant.
```suggestion
    public static final String COMMAND_WORD = "export";
```